src/route/index.ts

```diff
@@ -344,18 +352,22 @@ export async function initApiServer(
         url: "/token-details/:contractId",
         schema: {
           params: {
-            ["contractId"]: {
-              type: "string",
-              validator: (qStr: string) => isContractId(qStr),
+            type: "object",
+            required: ["contractId"],
+            properties: {
+              ["contractId"]: {
+                type: "string",
+                validator: (qStr: string) => isContractId(qStr),
+              },
             },
           },
           querystring: {
             type: "object",
             required: ["pub_key", "network"],
             properties: {
-              ["pub_key"]: {
+              ["pubKey"]: {
```

Comment:
does the casing change matter here?

Reply:
oh hey thank you yes it does, updating.

Reply:
fixed in 5175ffe1c52ed46a45c7fce926588f7649e9e158